### PR TITLE
update closed refs to os refs

### PR DESCRIPTION
**Pull request by ACoolmanTelicent**
# Work done
- telicent-io chagned to telicent-oss
- (optimistic) tweaks to Readme

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -17,7 +17,7 @@
     "@fortawesome/react-fontawesome": "^0.2.0",
     "@mui/material": "^5.15.10",
     "@tanstack/react-query": "^5.18.0",
-    "@telicent-io/ontologyservice": "^0.0.14-rc36",
+    "@telicent-oss/ontologyservice": "^0.0.14-rc36",
```
Comment (tatelicent) [inline package.json:20]: Are these version correct with the public packages?

File: package.json [hunk 2]
```diff
@@ -48,7 +48,7 @@
   },
   "peerDependencies": {
     "@tanstack/react-query": "^5.18.0",
-    "@telicent-io/ontologyservice": "^0.0.14-rc36",
+    "@telicent-oss/ontologyservice": "^0.0.14-rc36",
```
Comment (tatelicent) [inline package.json:51]: same for this one

File: yarn.lock [hunk 3]
```diff
@@ -4237,17 +4237,17 @@
   dependencies:
     "@tanstack/query-core" "5.24.1"
 
-"@telicent-io/ontologyservice@^0.0.14-rc36":
+"@telicent-oss/ontologyservice@^0.0.14-rc36":
   version "0.0.14-rc49"
-  resolved "https://npm.pkg.github.com/download/@telicent-io/ontologyservice/0.0.14-rc49/2bbcaec531ec195113437e62ff440fe7a35392af#2bbcaec531ec195113437e62ff440fe7a35392af"
+  resolved "https://npm.pkg.github.com/download/@telicent-oss/ontologyservice/0.0.14-rc49/2bbcaec531ec195113437e62ff440fe7a35392af#2bbcaec531ec195113437e62ff440fe7a35392af"
```
Comment (tatelicent) [inline yarn.lock:4242]: This doesnt look like it is using the public package

**Review by tatelicent — APPROVED** [review]
LGTM

**Comment by ACoolmanTelicent** [comment]
yep

**Review by davec504 — APPROVED** [review]

**Pull request merged**